members/ranks/daoists.md

```diff
@@ -1,14 +1,16 @@
 # DAOists
 
-DAOists are the caretakers of the server! They have multiple roles, each with their own unique blend of values, principles, skills and ideas. Different people are here for different reasons. Some are here for neutrality or to balance discussions, some to provide guidance whether it be technical or spiritual motivation, some to spread their knowledge and educate others, some to build things they believe in and support their vision for the future, some to connect everyone in DOTSAMA and so on.
+DAOists are the caretakers of the server. They have multiple roles, each with their own unique blend of values, principles, skills and ideas. Different people are here for different reasons. Some are here for neutrality or to balance discussions, some to provide guidance whether it be technical or spiritual motivation, some to spread their knowledge and educate others, some to build things they believe in and support their vision for the future, some to connect everyone in DOTSAMA and so on.
 
 The key is that DAOists represent the members of their class with the most integrity, skill or whatever else that is valuable about them. Most importantly they are trustworthy in terms of their loyalty to the DAO and generally tempered enough that in a tough situation they would always do their best to minimize risk to the DAO and its members.
 
 However since we are all humans and nobody is perfect, there will be many mistakes along the way that eventually will help us learn from them and drive us even further. DAOists want genuinely to see the DAO prosper and have strong enough conviction in that desire to not let it waver easily in the face of adversity.
 
 DAOists are selected from the most active ChaosDAO members who have demonstrated useful organizational skills, mature trustworthy behavior, and dedication to the Polkadot community.
 
-There are currently between 20-23 DAOists who form the governing body of ChaosDAO.
+There are currently between 18-23 DAOists who form the governing body of ChaosDAO.
+
+While many aspects of governance is now being opened up to include both DAOists and Regulars, some sensitive topics will still be reserved for DAOist discussion and voting only.
```

Comment:
is needs to be "are" in this opening line (aspects being the noun, whereas governance is the object of the preposition)

Reply:
👍